js/services/contact_service.js

```diff
@@ -46,6 +45,16 @@ angular.module('contactsApp')
 							});
 						}).then(function (contacts_) {
 							contacts_.map(function (contact) {
+								// Validate some fields
+								contact.validate('rev');
+								contact.validate('prodid');
+								contact.validate('version');
```

Comment:
I would suggest to move these three validations to a `contact.fix()` method, which returns whether anything has changed an the contact has to be updated. This way we know where to find it, when we want to add other stuff that should be automatically fixed.
What do you think?

Reply:
I'm OK with this :)